src/pages/Home.jsx

```diff
@@ -1,91 +1,141 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
 import faker from 'faker'
 import React from 'react'
+import { useState } from 'react'
+import { useQueryClient, useQuery } from 'react-query'
+import axios from 'axios'
+import classNames from 'classnames'
 
 function Home() {
+  const PAGE_LIMIT = 5
+  const [offset, setOffset] = useState(0)
+  const [selectedTag, setSelectedTag] = useState(null)
+  const queryClient = useQueryClient()
+  let params = {
+    offset: offset * PAGE_LIMIT,
+    limit: PAGE_LIMIT
+  }
```

Comment:
```suggestion
  const params = {
    offset: offset * PAGE_LIMIT,
    limit: PAGE_LIMIT,
    tag: selectedTag
  }
```
I think this will give you the same result.